gameServer.js

```diff
@@ -101,7 +102,8 @@ app.use('*', function(request, response){
 })
 
 function runCountdown() {
-    io.emit('start game', "players reached");
+  if (!game_started) io.emit('start game', "players reached");
+  game_started = true;
```

Comment:
Skyler, 
I believe that we want players to be able to join before and while the countdown runs but we do not want players to join after the countdown in complete. This is at least how other battle royal games are run.
To rephrase:
When x number of players join the lobby the countdown starts. As the countdown runs, up to y number of players are allowed to join (y > x). Once the countdown hits 0 no more people are allowed to join the game. x is our game minimum and y is our game maximum.

Reply:
Exactly. And the way the code is written right now, only x numbers of players can join because as soon as player number x joins, the bool gets flipped so the y number of players are not able to join during the countdown.

Reply:
mmm thats right.  So I also realized something else, the client is currently in charge of the countdown, meaning any time anyone joins their countdown starts from 10.  I will change that so the server is in charge